rts/lra/jaxrs/src/main/java/io/narayana/lra/filter/ServerLRAFilter.java

```diff
@@ -79,11 +80,13 @@
 import static org.eclipse.microprofile.lra.annotation.ws.rs.LRA.Type.NESTED;
 
 @Provider
+@ApplicationScoped
```

Comment:
@mmusgrov no the @Provider only implies CDI bean defining annotation in Quarkus. This doesn't work in WFLY because it's not spec mandated. So this annotation is actually needed to provide injections of LRAParticipantRegistry.

Reply:
btw, well spotted, this is the only change that is actually needed for Participant TCK tests (everything else Context TCK tests and threading issue).

Reply:
The RESTEasy documentation [1] says "If a JAX-RS Provider or javax.ws.rs.Application subclass does not define a scope explicitly, it is bound to the Application scope."

BTW Are you suggesting that the only way to @Inject CDI beans in WildFly is to make the filter ApplicationScoped?

[1] https://docs.jboss.org/resteasy/docs/3.0.2.Final/userguide/html/CDI.html#d4e2011

Reply:
In WFLY, we now run on Jakarta 10 which bundles CDI 4.0 which changed the default scanning to "annotated" so this is why the @Provider is no longer scanned and thus it is not a CDI bean unless annotated with bean-defining annotation.

Reply:
That's a thorough answer, appreciated.